src/submit.js

```diff
@@ -0,0 +1,14 @@
+import leaderBoard from './leaderBoard.js';
+import render from './render.js';
+
+export default async function submit(userName, userScore) {
```

Comment:
- [ ] Kindly use  arrow functions instead of the `function` keyword as this is part of the project requirements 👍🏿

Reply:
@prolajumokeoni  Thank you for your comment! I will do 💯